v2/datastream-to-sql/src/main/java/com/google/cloud/teleport/v2/templates/DataStreamToSQL.java

```diff
@@ -260,6 +260,17 @@ public interface Options extends PipelineOptions, StreamingOptions {
     int getNumThreads();
 
     void setNumThreads(int value);
+
+    @TemplateParameter.Integer(
```

Comment:
should there be a default here?

Reply:
I think better not change the default behaviour. This is why I do not set any default value.